osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -162,6 +162,9 @@ public double SliderVelocityMultiplier
         [JsonIgnore]
         public SliderTailCircle TailCircle { get; protected set; }
 
+        [JsonIgnore]
+        public SliderRepeat LastRepeat { get; protected set; }
```

Comment:
Should be annotated as `[CanBeNull]`, because contrary to `{Head,Tail}Circle`, this can feasibly be null on any given slider (if it has no repeats).

And yes I'm saying to use jetbrains attributes for now because changing this to *not* use jetbrains attributes likely bloats this diff with 50 files of unrelated NRT changes.